pkg/kube/cluster-init.sh

```diff
@@ -68,6 +68,43 @@ setup_prereqs () {
         check_network_connection
 }
 
+check_start_containerd() {
+        # Needed to get the pods to start
+        if [ ! -L /usr/bin/runc ]; then
+                ln -s /var/lib/rancher/k3s/data/current/bin/runc /usr/bin/runc
+        fi
+        if [ ! -L /usr/bin/containerd-shim-runc-v2 ]; then
+                ln -s /var/lib/rancher/k3s/data/current/bin/containerd-shim-runc-v2 /usr/bin/containerd-shim-runc-v2
+        fi
+
+        if pgrep -f "containerd --config" >> $INSTALL_LOG 2>&1; then
+                logmsg "k3s-containerd is alive"
+        else
+                logmsg "Starting k3s-containerd"
+                mkdir -p /run/containerd-user
+                nohup /var/lib/rancher/k3s/data/current/bin/containerd --config /etc/containerd/config-k3s.toml &
```

Comment:
since you are using `nohup` here, I think it's better to redirect the output to some log file, otherwise it will append to nohup.out, which can mix with the k3s process also being started using nohup...

Reply:
@rene The k3s process has a config file directing all its logging to /var/lib/rancher/k3s/k3s.log instead.